src/graphql/helpers.ts

```diff
@@ -1,6 +1,13 @@
-import { getAddress } from '@ethersproject/address';
+import {
+  getFormattedAddress,
+  isEvmAddress,
+  isStarknetAddress
+} from '@snapshot-labs/snapshot.js/src/utils';
```

Comment:
I think we should import like everywhere
![Untitled 5](https://github.com/snapshot-labs/snapshot-hub/assets/15967809/89636640-2fac-4ab6-afcf-2f1e794acace)

Reply:
Weird, I don't have this error on the console. What version of node are you using ?